### PR TITLE
Unhinged updated costs

### PR DESCRIPTION
**Pull request by NBKelly**
This takes into account the updated costs
See #7347

### CODE REVIEW

File: src/clj/game/cards/unhinged.clj [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+                                                                                   (continue-ability
+                                                                                     state side
+                                                                                     (reveal-fn)
+                                                                                     card nil)
+                                                                                   (effect-completed state side eid))))}
+                                            :no-ability {:msg (msg "decline to play " (:title top-card))}}}
+                                          card nil)
+                                        ;; installable
+                                        (if (can-pay? state side (assoc eid :source card :source-type :runner-install) top-card nil
+                                                      (install-cost state side top-card))
```
[the remark this answers is not in the document]
Comment (NBKelly) [reply src/clj/game/cards/unhinged.clj:156]: (install-cost ...) already returns a (->c) function, right?

[the remark this answers is not in the document]
Comment (NBKelly) [reply src/clj/game/cards/unhinged.clj:156]: oh no, that's runner-install-cost (which I should be using)

**Comment by NBKelly** [comment]
I updated this to take the standard+ format out of visibility. We can merge this, or scrap the whole thing, I don't mind either way (but if we scrap it, we should remove the cards from netrunner-data).

File: src/clj/game/core/engine.clj [hunk 2]
```diff
@@ -1060,10 +1060,12 @@
   [state _ eid]
   (let [corp-uniques (get-old-uniques state :corp)
         runner-uniques (get-old-uniques state :runner)
-        consoles (->> (get-in @state [:runner :rig :hardware])
+	consoles (->> (get-in @state [:runner :rig :hardware])
```
Comment (NoahTheDuke) [inline src/clj/game/core/engine.clj:1063]: spacing

File: src/clj/game/core/ice.clj [hunk 3]
```diff
@@ -320,6 +320,7 @@
 (defn get-strength
   [card]
   (when (or (ice? card)
+            (:strength card)
```
Comment (NoahTheDuke) [inline src/clj/game/core/ice.clj:323]: how does this apply?

File: src/clj/web/lobby.clj [hunk 4]
```diff
@@ -44,7 +44,7 @@
      :room room
      :save-replay save-replay
      :spectatorhands spectatorhands
-     :singleton (when (some #{format} `("standard" "startup" "casual" "eternal")) singleton)
+     :singleton (when (some #{format} `("standard" "startup" "standard-plus" "casual" "eternal")) singleton)
```
Comment (NoahTheDuke) [inline src/clj/web/lobby.clj:47]: remove this?

Comment (NoahTheDuke) [reply src/clj/web/lobby.clj:47]: and the rest of the references?

**Review by NoahTheDuke — CHANGES_REQUESTED** [review]

File: src/clj/game/core/ice.clj (hunk 3 above)
Comment (NBKelly) [reply src/clj/game/core/ice.clj:323]: It means we can do forced encounters with non-ice cards, so long as they have a listed strength.
There's an upgrade that pretends to be an ice on access, and it's surprising that this is the *only* change needed to make it work.

File: src/clj/game/core/engine.clj [hunk 5]
```diff
@@ -1064,6 +1064,8 @@
                       (filter console?))
         consoles (when (< 1 (count consoles))
                    (butlast consoles))
+	consoles (when-not (any-effects state :runner :enable-multiple-consoles)
```
Comment (NBKelly) [inline src/clj/game/core/engine.clj:1067]: I *should* make a command to enable this too (a job for later)

File: src/clj/web/lobby.clj (hunk 4 above)
Comment (NBKelly) [reply src/clj/web/lobby.clj]: I *think* that's everything

File: src/clj/game/core/ice.clj (hunk 3 above)
Comment (NoahTheDuke) [reply src/clj/game/core/ice.clj:323]: in that case, can you make it more specific? either by name or by "upgrade and :strength"?

File: src/clj/web/lobby.clj (hunk 4 above)
Comment (NoahTheDuke) [reply src/clj/web/lobby.clj]: thanks homie

File: src/clj/game/core/ice.clj (hunk 3 above)
Comment (NBKelly) [reply src/clj/game/core/ice.clj]: I added a comment which explains the usage. We can get the (printed) strength of a card if it's ice, a program, or has a printed strength.

**Pull request closed without merging**